ocp_resources/machine_config_pool.py

```diff
@@ -2,8 +2,70 @@
 
 
 class MachineConfigPool(Resource):
+    """
+    MachineConfigPool object. API reference:
+    https://docs.openshift.com/container-platform/4.12/rest_api/machine_apis/machineconfigpool-machineconfiguration-openshift-io-v1.html
+
+    Args:
+        name (string): Object name.
+        machine_config_selector (object): List of matching labels, to determine on which nodes to apply this
```

Comment:
machineConfigSelector -> matchLabels get a dict of label_name; label_value
```
  machineConfigSelector:
    matchLabels:
      machineconfiguration.openshift.io/role: master
```
matchExpressions gets a list of dicts

Reply:
Ack